lab_6_pipeline/pipeline.py

```diff
@@ -18,22 +45,58 @@ def __init__(self, path_to_raw_txt_data: Path):
         """
         Initializes CorpusManager
         """
+        self._storage = {}
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._validate_dataset()
 
     def _validate_dataset(self) -> None:
         """
         Validates folder with assets
         """
+        #do not use os, pathlib only
+        #glob can be used
+        texts = list(self.path_to_raw_txt_data.glob('*.txt'))
+        meta_info = list(self.path_to_raw_txt_data.glob('*.json'))
+        right_texts = sorted([int(re.match(r'\d+', i.name).group()) for i in texts])
+
+        if not self.path_to_raw_txt_data.exists():
```

Comment:
you start checking path after you have made traversal through its content - why?

Reply:
I changed the order of the checks, the previous one was actually illogical